lib/ansible/executor/playbook_executor.py

```diff
@@ -265,7 +265,7 @@ def run(self):
         if context.CLIARGS['start_at_task'] and not self._tqm._start_at_done:
             display.error(
                 "No matching task \"%s\" found."
-                " Note: --start-at-task can only follow static includes."
+                " Note: --start-at-task can only follow dynamic includes."
```

Comment:
This is not really correct but the current wording is confusing (`git blame`'ed to my commit ;-) ).

I believe by "following static includes" it is meant that `--start-at-task` can target tasks **_within_** static includes.

So the message should be wordsmithed to contain the following information, `--start-at-task`:

- can target tasks **_within_** static includes
- cannot target tasks **_within_** dynamic includes
- can target the dynamic include task itself
- cannot target the static include task itself

Reply:
To explain the above, static includes (imports) are removed and replaced with the imported tasks when compiling the playbook so by the time you get here, they don't exist, but the imported tasks do exist.

On the contrary dynamic includes (include_X) exist at this point and have not been processed yet, so the 'future included tasks' are not accessible but the include itself is.